### ISSUE TITLE
Unhandled JSDisconnectedException

### ISSUE THREAD

**Issue by garrynewman** [apexcharts/Blazor-ApexCharts#253]
Using Blazor Serverside and getting these exceptions every now and then in our backend.

```
Microsoft.JSInterop.JSDisconnectedException: JavaScript interop calls cannot be issued at this time. This is because the circuit has disconnected and is being disposed.
  File "RemoteJSRuntime.cs", line 107, in void RemoteJSRuntime.BeginInvokeJS(long asyncHandle, string identifier, string argsJson, JSCallResultType resultType, long targetInstanceId)
  File "JSRuntime.cs", line 140, in ValueTask<TValue> JSRuntime.InvokeAsync<TValue>(long targetInstanceId, string identifier, CancellationToken cancellationToken, object[] args)
  File "JSRuntime.cs", line 95, in async ValueTask<TValue> JSRuntime.InvokeAsync<TValue>(long targetInstanceId, string identifier, object[] args)
  File "JSRuntimeExtensions.cs", line 30, in async ValueTask JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime jsRuntime, string identifier, params object[] args)
  ?, in async void ApexChart<TItem>.Dispose()+(?) => { }

```

I understand that the jsinterop stuff doesn't really have any way to tell if it's connected or not, but maybe these could be wrapped and handled?

**Comment by joadan** [comment]
Hello,

Not sure how to handle it? if js is disconected not much we can do or?
Sorry If I'm missing something obvious here.

Thank you

**Comment by garrynewman** [comment]
You're right - I was going to suggest try/catch and swallowing it.. but we might as well do that on our side.

**Comment by haas-daniel** [comment]
Hi, IMHO this exception should be catched by the component.

This is also suggested by Microsoft: https://learn.microsoft.com/en-us/aspnet/core/blazor/javascript-interoperability/?view=aspnetcore-7.0#javascript-interop-calls-without-a-circuit

**Comment by joadan** [comment]
Hi, 

Ok I get it it's about handling the exception when the component is being disposed.

**Comment by haas-daniel** [comment]
Yes, exactly

**Comment by joadan** [comment]
Fixed in v1.3.0
